docs/01-overview/02-main-areas/service-management/smgt-11-hb-create-addons.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Create addons
+---
+
+Addons which the Helm Broker uses must have a specific structure. These are all possible files that you can include in your addons:
+
+```text
+sample-addon/
+   ├── meta.yaml                             # [REQUIRED] A file which contains metadata information about this addon
+   ├── chart/                                # [REQUIRED] A directory which contains a Helm chart that installs your Kubernetes resources
+   │    └── {chart-name}/                    # [REQUIRED] A Helm chart directory
+   │         └── ....                        # [REQUIRED] Helm chart files
+   ├── plans/                                # [REQUIRED] A directory which contains the possible plans for an installed chart
+   │    ├── example-enterprise               # [REQUIRED] A directory which contains files for a specific plan
+   │    │   ├── meta.yaml                    # [REQUIRED] A file which contains metadata information about this plan
+   │    │   ├── bind.yaml                    # A file which contains information required to bind this plan
+   │    │   ├── create-instance-schema.json  # JSON schema definitions for creating a ServiceInstance
+   │    │   ├── bind-instance-schema.json    # JSON schema definitions for binding a ServiceInstance
+   │    │   ├── update-instance-schema.json  # JSON schema definitions for updating a ServiceInstance
+   │    │   └── values.yaml                  # Default configuration values in this plan for a chart defined in the `chart` directory
+   │    └── ....
+   │
+   └── docs/                                 # A directory which contains documentation for this addon
+        ├── meta.yaml                        # [REQUIRED] A file which contains metadata information about documentation for this addon
+        ├── {assets}                         # Files with documentation and assets
+        └── ....
+```
+
+> **NOTE:** All file names in an addon repository are case-sensitive.
+
+For details about particular files, read the following sections.
+
+## meta.yaml file
+
```

Comment:
aren't those tables more a kind of technical reference information that users look up occasionally?

Reply:
Actually, those tables describe the files a repository must have to be considered an addon. If someone wants to create an addon, this info is essential so I won't say it's a technical reference 🤔 Rather a tutorial? 

`To create an addon, create a repository that has these files specified.`